### ISSUE TITLE
Scope is not detected correctly after copy-pasting pieces of code

### ISSUE THREAD

**Issue by yamalight** [tvooo/scope-inspector#7]
So, as the title says - scope is not detected and highlighted properly if you copy-paste some pieces of code.

Example that always shows scope as global:
Step 1. Create new file with two functions like so:

``` js
var test = function() {
    var obj = [];
};

var test2 = function() {
    var anotherObj = [];
};
```

Step 2. Paste following code somewhere in the function (doesn't matter which one):

``` js
    code
    .doStuff()
    .stop();
```

Step 3. See how it messed up the scope detection
Step 4. Save file - now scopes are fine again
Step 5. Remove pasted code
Step 6. Scope is messed up once again
Step 7. Save file - now scopes are back to normal

It seems like scope is not re-evaluated on docs change (or not always?) but only on save.
